pkg/expapi/register.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package expapi
+
+import (
+	// api/latest.init creates a RESTMapper using the objects currently registered.
+	// We need to register our objects after api/latest is initialized so that the
+	// RESTMapper for /api does not include experimental objects.
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
```

Comment:
As per discussion with @uluyol , this can be overcome by adding the following condition to api/latest/latest.go

```!strings.Contains(oType.PkgPath(), "kubernetes/pkg/api")```